src/qibo/models/error_mitigation.py

```diff
@@ -9,9 +9,16 @@
 from scipy.optimize import curve_fit
 
 from qibo import gates
-from qibo.backends import _check_backend, _check_backend_and_local_state, get_backend
+from qibo.backends import (
+    NumpyBackend,
+    _check_backend,
+    _check_backend_and_local_state,
+    get_backend,
+)
 from qibo.config import raise_error
 
+SIMULATION_BACKEND = NumpyBackend()
```

Comment:
I leave this as a comment here, and if we want we can open a dedicated issue. 
As discusse wit @BrunoLiegiBastonLiegi, this is a solution for now, but we would ideally want to have:

- clifford backend for pure clifford methods (e.g. importance clifford sampling);
- quasi clifford backend for hybrid solutions (e.g. CDR or vnCDR).